src/StylelintError.js

```diff
@@ -3,7 +3,7 @@ class StylelintError extends Error {
    * @param {string=} messages
    */
   constructor(messages) {
-    super(messages);
+    super(`\u001b[1;31m[stylelint]\u001b[1;0m ${messages}`);
```

Comment:
Make sense to respect `stats.colors` option, because some developers disable colors